### PR TITLE
Add docs section for using S3 with Spark

### PR DESCRIPTION
**Pull request by dimas-b**

### CODE REVIEW

File: site/docs/guides/spark-s3.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Accessing data in S3 with Spark
```
Comment (ajantha-bhat) [inline site/docs/guides/spark-s3.md:1]: Overall good document. Thanks for adding.

May be we should add a link to this in `https://projectnessie.org/tools/iceberg/spark/` . Right now it kind of looks hidden in guides section.

File: site/docs/guides/spark-s3.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Setting up Nessie Server
+
+Start the Nessie server container from the `projectnessie/nessie` Docker image in default mode.
+
+```shell
+docker run -p 19120:19120 projectnessie/nessie:latest
+```
+
+Note: this example with run the Nessie Server using in-memory storage for table metadata. If/when the container
```
Comment (nastra) [inline site/docs/guides/spark-s3.md:16]: with -> will

File: site/docs/guides/spark-s3.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+is deleted Nessie data about table changes will be lost, yet the data files in S3 will remain.
```
Comment (nastra) [inline site/docs/guides/spark-s3.md:17]: missing a comma here: ...is deleted, Nessie data about...

File: site/docs/guides/spark-s3.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+Note: The above example uses the `spark-sql` shell, but the same configuration options apply to `spark-shell`.
+
+# Authentication
+
+This example uses implicit AWS authentication via credentials configured in a credentials file plus the `AWS_PROFILE`
+environment variable.
+
+The Nessie Server in this example does not require authentication.
+
+If the Nessie Server runs with authentication enabled additional configuration parameters will be required in the
```
Comment (nastra) [inline site/docs/guides/spark-s3.md:90]: comma missing: ...with authentication enabled, additional configuration...

**Review by nastra — DISMISSED** [review]

File: site/docs/guides/spark-s3.md (hunk 3 above)
Comment (dimas-b) [reply site/docs/guides/spark-s3.md:17]: ah... thx... to long away from school ;)

**Pull request merged**